envs/monkey_zoo/blackbox/conftest.py

```diff
@@ -32,6 +32,11 @@ def no_gcp(request):
     return request.config.getoption("--no-gcp")
 
 
+@pytest.fixture(scope="session")
+def machines_to_start(request):
+    return request.config.getoption("-k")
```

Comment:
What does it do? Get's the "-k" options provided to pytest?

Reply:
Yep